etc/kubernetes/helm/templates/ingress.yaml

```diff
@@ -0,0 +1,24 @@
+{{ if .Values.ingress.enabled }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  namespace: {{ .Values.ingress.namespace }}
```

Comment:
Let's default this to the same namespace configured on the helm command line.  This is represented by `{{ .Release.Namespace }}`